flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/operations/SqlOtherOperationConverterTest.java

```diff
@@ -222,26 +226,30 @@ void testShowViews() {
         Operation operation = parse(sql);
         assertThat(operation).isInstanceOf(ShowViewsOperation.class);
 
-        ShowViewsOperation showTablesOperation = (ShowViewsOperation) operation;
```

Comment:
Nice!  I missed this typo...  Very natural given that one should be using the other PR to make this one!